test/contrib/salesforce_test.py

```diff
@@ -88,7 +89,11 @@ def soql(self):
 
 
 class TestSalesforceQuery(unittest.TestCase):
-    @mock.patch('__builtin__.open', side_effect=mocked_open)
+    patch_name = '__builtin__.open'
+    if sys.version_info.major > 2:
+        patch_name = 'builtins.open'
+
+    @mock.patch(patch_name, side_effect=mocked_open)
```

Comment:
Not sure if I can use it to import a generic builtins, but I can at least use it to check for the Python version. 

The challenge here is the `patch_name` string gets interpreted further down the stack and I'm not actually importing builtins, so flake8 throws an error.